AppManager/app_manager_server.py

```diff
@@ -203,8 +218,56 @@ def start_app(config):
     monit_interface.stop(watch)
     return BAD_PID
 
+  if 'log_size' in config.keys():
+    log_size = config['log_size']
+  else:
+    if config['app_name'] == APPSCALE_DASHBOARD_ID:
+      log_size = TEN_MB
```

Comment:
A parameter through config will override all other defaults.
At this stage I think it's the best option.